src/test/BiscuitDisplayerTest.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace BiscuitApp\Test;
+class BiscuitDisplayerTest
+{
+
+}
```

Comment:
for these test files you need to extend TestCase (PHPUnit class) and you need to import that class. do that in all 3 files